frame/fast-unstake/src/lib.rs

```diff
@@ -142,7 +142,7 @@ pub mod pallet {
 
 		/// Use only for benchmarking.
 		#[cfg(feature = "runtime-benchmarks")]
-		type MaxBackersPerValidator: Get<u32>;
+		type MaxExposurePageSize: Get<u32>;
```

Comment:
Still think this is part of `StakingInterface`.

Reply:
Removed 👍